generators/client/templates/src/main/webapp/app/_app.constants.js

```diff
@@ -1,7 +1,9 @@
-"use strict";
-// DO NOT EDIT THIS FILE, EDIT THE GULP TASK NGCONSTANT SETTINGS INSTEAD WHICH GENERATES THIS FILE
-angular.module('<%=angularAppName%>')
+(function() {
+  "use strict";
+  // DO NOT EDIT THIS FILE, EDIT THE GULP TASK NGCONSTANT SETTINGS INSTEAD WHICH GENERATES THIS FILE
+  angular.module('<%=angularAppName%>')
```

Comment:
this file will be overwritten by the gulp ngConstants task, so the changes needs to be updated there as well https://github.com/jhipster/generator-jhipster/blob/master/generators/client/templates/gulpfile.js#L264